.github/workflows/build-cli.yml

```diff
@@ -5,6 +5,9 @@
 on:
   workflow_call:
     inputs:
+      version:
+        required: true
+        type: string
```

Comment:
i think this would break the canary, that re-uses the build-cli https://github.com/block/goose/blob/main/.github/workflows/canary.yml#L22-L23

Reply:
on the canary note, do we want to add a version suffix there as well?